go/dhcp/utils.go

```diff
@@ -283,4 +296,26 @@ func ExcludeIP(dhcpHandler *DHCPHandler, ip_range string) {
 			dhcpHandler.available.ReserveIPIndex(uint64(position), FakeMac)
 		}
 	}
+	return excludeIPs
+}
+
+// Assign static IP address to a mac address and remove it from the pool
+func AssignIP(dhcpHandler *DHCPHandler, ip_range string) (map[string]uint32, []net.IP) {
+	couple := make(map[string]uint32)
+	var iplist []net.IP
+	if ip_range != "" {
+		rgx, _ := regexp.Compile("((?:[0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}):((?:[0-9]{1,3}.){3}(?:[0-9]{1,3}))")
+		iprange := strings.Split(ip_range, ",")
```

Comment:
Having ip_range (soon to be ipRange) and iprange variables in the same scope makes it confusing and error-prone while reading the code.

I'd suggest calling this ipRangeArray or something that is clear that you're not dealing with the string variable